core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/service/ReactorFlowManager.java

```diff
@@ -176,8 +178,10 @@ public void triggerClusterUpscale(Long stackId, HostGroupAdjustmentJson hostGrou
     public void triggerClusterDownscale(Long stackId, HostGroupAdjustmentJson hostGroupAdjustment) {
         String selector = FlowChainTriggers.FULL_DOWNSCALE_TRIGGER_EVENT;
         ScalingType scalingType = hostGroupAdjustment.getWithStackUpdate() ? ScalingType.DOWNSCALE_TOGETHER : ScalingType.DOWNSCALE_ONLY_CLUSTER;
+        int adjustment = Optional.ofNullable(hostGroupAdjustment.getScalingAdjustment())
+                .orElseThrow(() -> new BadRequestException("Unable to define hostgroup scaling adjustment amount"));
         Acceptable event = new ClusterAndStackDownscaleTriggerEvent(selector, stackId,
-                hostGroupAdjustment.getHostGroup(), hostGroupAdjustment.getScalingAdjustment(), scalingType);
+                hostGroupAdjustment.getHostGroup(), adjustment < 0 ? adjustment * -1 : adjustment, scalingType);
```

Comment:
I think the Math.abs(adjustment) is much better.

Reply:
good idea, going to change it